phenol-annotations/src/main/java/org/monarchinitiative/phenol/annotations/formats/hpo/HpoDiseaseAnnotationDefault.java

```diff
@@ -38,7 +35,7 @@ static HpoDiseaseAnnotationDefault of(TermId termId, Collection<HpoDiseaseAnnota
     // 2. Process ratio and observation intervals in a single loop
     for (HpoDiseaseAnnotationMetadata datum : metadata) {
       // Ratio
-      Optional<Ratio> ratio = datum.frequency().ratio();
+      Optional<Ratio> ratio = datum.frequency().flatMap(AnnotationFrequency::ratio);
```

Comment:
I think that every annotation should always have a Ratio, and that if there is no data in phenotype.hpoa, then the implementing class should decide what the ratio is (I think it should be 1/1 for our DefaultAnnotation class). This would make it a lot easier to use the code. We could also have a boolean method such as
```
boolean hasExplicitFrequencyAnnotation()
```

Reply:
@pnrobinson OK, let's make `ratio` non-null attribute of the annotation. This means we'll parameterize the HPOA loading, as we're doing right now.
Let me deal with this in a separate issue.